STYLE_GUIDE.md

```diff
@@ -391,6 +391,15 @@ end
   output.>append(file_name).>append(":").>append(msg)
   ```
 
+- Explicit partial calls are postfixed with '?' operator without whitespace inbetween.
```

Comment:
ironically, I think "whitespace inbetween" should be "whitespace in between". :grinning: